site/realm/data_sources/mongodb-atlas/aiidprod/classifications/rules.json

```diff
@@ -7,33 +7,118 @@
             "apply_when": {
                 "%%user.custom_data.roles": "admin"
             },
+            "read": true,
             "write": true,
             "insert": true,
             "delete": false,
-            "search": true,
-            "additional_fields": {}
+            "search": true
         },
         {
             "name": "taxonomy_editor",
             "apply_when": {
                 "%%user.custom_data.roles": "taxonomy_editor"
             },
+            "read": true,
             "write": true,
             "insert": true,
             "delete": false,
-            "search": true,
-            "additional_fields": {}
+            "search": true
         },
         {
-            "name": "taxonomy_editor_cset",
+            "name": "taxonomy_editor_csetv0",
```

Comment:
There was a plan in the future to have taxonomies dynamically created, so this won't be possible, but it works in the meantime.

Reply:
Yes, I did the `realm pull` before pushing.
Regarding the dynamic taxonomies creation, we should think of another approach in the future since we can't modify these `rules.json` files dynamically.